src/test/java/org/tikv/raw/RawKVClientTest.java

```diff
@@ -307,6 +327,76 @@ public void deleteRangeTest() {
     client.deleteRange(ByteString.EMPTY, ByteString.EMPTY);
   }
 
+  @Test
+  public void batchDeleteTest() {
+    int cnt = 8;
+    List<ByteString> keys = new ArrayList<>();
+    for (int i = 0; i < cnt; i++) {
+      ByteString key = getRandomRawKey().concat(ByteString.copyFromUtf8("batch_delete_test"));
+      client.put(key, key);
+      keys.add(key);
+    }
+
+    client.batchDelete(keys);
+
+    for (int i = 0; i < cnt; i++) {
+      checkNotExist(keys.get(i));
+    }
+  }
+
+  @Test
+  public void scan0test() {
+    int cnt = 8;
+    ByteString prefix = ByteString.copyFromUtf8("scan0_test");
+    client.deletePrefix(prefix);
+    List<ByteString> keys = new ArrayList<>();
+    for (int i = 0; i < cnt; i++) {
+      ByteString key = prefix.concat(getRandomRawKey());
+      client.put(key, key);
+      keys.add(key);
+    }
+
+    int i = 0;
+    Iterator<KvPair> iter = client.scan0(prefix, ByteString.EMPTY, cnt);
+    while (iter.hasNext()) {
+      i++;
+      KvPair pair = iter.next();
+      assertEquals(pair.getKey(), pair.getValue());
+    }
+    assertEquals(cnt, i);
+
+    i = 0;
+    iter = client.scan0(prefix, ByteString.EMPTY, true);
```

Comment:
Scan0 is wired to be present in an API. After going through the code, I think you should test `RawKVClientBase` instead. Declare `RawKVClientTest.Client` to `RawKVClientBase` instead of `RawKVClient`